### ISSUE TITLE
bioconda version does not have --download

### ISSUE THREAD

**Issue by happykhan** [phac-nml/irida-linker#4]
Installing from conda does not seem to give the option for --download (or it doesn't work)

**Comment by happykhan** [comment]
bueller?

**Comment by tom114** [comment]
Hi @happykhan, I just did a fresh install of the linker from conda and don't see any issues downloading files with the `--download` option.  The files downloaded just fine from the IRIDA install I pointed to.  Can you give any more information about error messages or what you're seeing when it fails?  Thanks.

**Comment by happykhan** [comment]
```
ngsArchiveLinker.pl  -c  ~/.irida/ngs-archive-linker.conf   -p 6 --download --ignore –flat


Listing all samples from project 6

Error: Script cannot see a file to be linked: /qib/services/galaxy/irida/sequence/749/2/P3-5099_1.fastq.  Ensure you have access to the sequence files directory. at /miniconda3/bin/ngsArchiveLinker.pl line 242, <> line 2.
```



It seems to be installed fine:

```
 ngsArchiveLinker.pl -h

Usage:

    ngsArchiveLinker.pl -b <API URL> -p <projectId> -d <outputDirectory> [-s

    <sampleId> ...]
......
 `

**Comment by happykhan** [comment]
#3  might also be related. i dont know.

**Comment by tom114** [comment]
Hey again @happykhan .  It looks to me like you've got an old bugged version of the linker that had the file check regardless of the download option.  It looks like @apetkau fixed that in fc5b3726797efd1bca02c83a17642f30130fdfba.  Can you update your linker with `conda update irida-linker`?

**Comment by happykhan** [comment]
we got it working in the end by installing from source and it was fine. it its been addressed in current conda version that's fine.  just thought ya'll might need to bump the conda version.
